core/src/test/java/org/mskcc/cbio/portal/scripts/TestIntegrationTest.java

```diff
@@ -205,6 +206,20 @@ public void testLoadStudyEs0() throws Throwable {
             assertEquals(3, clinicalAttributes.size());
             clinicalAttributes = apiService.getPatientClinicalAttributes();
             assertEquals(4, clinicalAttributes.size());
+            List<DBClinicalSampleData> clinicalComputedSampleData = apiService.getSampleClinicalData("study_es_0", Arrays.asList("MUTATION_COUNT","FRACTION_GENOME_ALTERED"), Arrays.asList("TCGA-A2-A04P-01"));
+            Boolean mutationCountExists = false;
+            Boolean fractionGenomeAlteredExists = false;
+            for (DBClinicalSampleData dbClinicalSampleData: clinicalComputedSampleData) {
+                if (dbClinicalSampleData.attr_id.equals("MUTATION_COUNT")) {
+                    mutationCountExists = true;
+                    assertEquals("TCGA-A2-A04P-01 has one mutation in MUTATION_COUNT", "1", dbClinicalSampleData.attr_val);
+                } else if (dbClinicalSampleData.attr_id.equals("FRACTION_GENOME_ALTERED")) {
+                    fractionGenomeAlteredExists = true;
+                    assertEquals("TCGA-A2-A04P-01 has 0 FRACTION_GENOME_ALTERED", 0.0, Float.parseFloat(dbClinicalSampleData.attr_val), 0.01);
```

Comment:
This is new to me .. the "double is equal within a delta range" construction. But it looks like this is the preferred way, and it does make sense for issues related to precision.

Reply:
it actually gave me an error when doing a float comparison without using delta range, so that was nice